src/pcre2_jit_compile.c

```diff
@@ -7432,18 +7435,30 @@ DEFINE_COMPILER;
 
 sljit_emit_op_dst(compiler, SLJIT_FAST_ENTER, RETURN_ADDR, 0);
 
-OP2(SLJIT_SUB, TMP1, 0, TMP1, 0, SLJIT_IMM, 0x0a);
-OP2U(SLJIT_SUB | SLJIT_SET_LESS_EQUAL, TMP1, 0, SLJIT_IMM, 0x0d - 0x0a);
+#ifdef EBCDIC
+OP2U(SLJIT_SUB | SLJIT_SET_Z, TMP1, 0, SLJIT_IMM, CHAR_LF);
+OP_FLAGS(SLJIT_MOV, TMP2, 0, SLJIT_EQUAL);
```

Comment:
Can these be organized in some way to groups? Maybe a `static const sljit_u8` bitset could be better.

Reply:
Hmm. The ASCII code is basically `if (input >= '\n' && input <= '\r')`, and I've added an EBCDIC version which doesn't assume that the characters are consecutive, in the form `if (input == '\n' || input == '\v' || input == '\f' || input == '\r')`.

Would you really prefer a 32-byte bitset instead? It would be a bit of a pain to construct the bitset literal from the CHAR values. I'll happily do it if you say so.

I don't care about "performance", I only want to get the tests passing.